Sources/ArcGISToolkit/Components/AR/ARGeoView.swift

```diff
@@ -0,0 +1,384 @@
+            didSet {
+                // If we're already tracking, reset tracking to use the new configuration.
+                if isTracking, deviceSupportsARKit {
+                    arView.session.run(arConfiguration, options: .resetTracking)
+                }
+            }
+        }
+        
+        /// The last portrait or landscape orientation value.
+        var lastGoodDeviceOrientation = UIDeviceOrientation.portrait
+        
+        /// The tracking mode controlling how the locations generated from the location data source are used during AR tracking.
+        private var trackingMode: ARGeoViewTrackingMode = .ignore
+        
+        /// The initial transformation used for a table top experience.  Defaults to the Identity Matrix.
+        var initialTransformation: TransformationMatrix = .identity
+        
+        /// The data source used to get device location.  Used either in conjuction with ARKit data or when ARKit is not present or not being used.
+        var locationDataSource: LocationDataSource? {
+            didSet {
+                // locationDataSource?.locationChangeHandlerDelegate = self
+            }
+        }
+        
+        /// The translation factor used to support a table top AR experience.
+        var translationFactor: Double {
+            get { cameraController.translationFactor }
+            set { cameraController.translationFactor = newValue }
+        }
+        
+        /// Determines the clipping distance around the originCamera. The units are meters; the default is `nil`.
+        /// When the value is set to `nil`, there is no enforced clipping distance and therefore no limiting of displayed data.
+        /// Setting the value to 10.0 will only render data 10 meters around the origin camera.
+        var clippingDistance: Double? {
+            get { cameraController.clippingDistance }
+            set { cameraController.clippingDistance = newValue }
+        }
+        
+        /// The viewpoint camera used to set the initial view of the sceneView instead of the device's GPS location via the location data source.  You can use Key-Value Observing to track changes to the origin camera.
+        var originCamera: Camera {
+            get { cameraController.originCamera }
+            set { cameraController.originCamera = newValue }
+        }
+        
+        /// The `TransformationMatrixCameraController` used to control the Scene.
+        var cameraController: TransformationMatrixCameraController {
+            sceneViewController.cameraController as! TransformationMatrixCameraController
+        }
+        
+        var sceneViewController: ArcGISSceneViewController
+        
+        public init(
+            scene: ArcGIS.Scene,
+            cameraController: TransformationMatrixCameraController,
+            graphicsOverlays: [GraphicsOverlay] = [],
+            analysisOverlays: [AnalysisOverlay] = [],
+            renderVideoFeed: Bool = true
```

Comment:
```suggestion
            analysisOverlays: [AnalysisOverlay] = [],
            trackingMode: ARGeoViewTrackingMode = .ignore,
            renderVideoFeed: Bool
```
The `ARGeoView` initializer sets the tackingMode, so that value should be passed here as well. Also `renderVideoFeed` does not have a default argument in the `ARGeoView` initializer, so that default argument should be removed here.

Reply:
parameters with defaults should be last.